qemu/tests/uefi_check_secure_MOR.py

```diff
@@ -0,0 +1,76 @@
+import logging
+import os
+
+from virttest import env_process
+from virttest import data_dir
+
+
+def run(test, params, env):
+    """
+    Please make sure the guest installed with signed driver
+    Verify Secure MOR control feature using Device Guard tool in Windows guest:
+
+    1) Boot up a guest.
+    2) Check if Secure Boot is enable.
+    3) Download Device Guard and copy to guest.
+    4) Enable Device Guard and check the output.
+    5) Reboot guest.
+    5) Run Device Guard and check the output.
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+
+    def execute_powershell_command(command, timeout=30):
+        status, output = session.cmd_status_output(command, timeout)
+        if status != 0:
+            test.fail("execute command fail: %s" % output)
+        return output
+
+    login_timeout = int(params.get("login_timeout", 360))
+    params["ovmf_vars_filename"] = 'OVMF_VARS.secboot.fd'
+    params["cpu_model_flags"] = ',hv-passthrough'
+    params["start_vm"] = 'yes'
+    env_process.preprocess_vm(test, params, env, params['main_vm'])
+    vm = env.get_vm(params["main_vm"])
+    session = vm.wait_for_serial_login(timeout=login_timeout)
+
+    check_cmd = params['check_secure_boot_enabled_cmd']
+    dgreadiness_path_command = params['dgreadiness_path_command']
+    executionPolicy_command = params['executionPolicy_command']
+    enable_command = params['enable_command']
+    ready_command = params['ready_command']
+    try:
+        output = session.cmd_output(check_cmd)
+        if 'False' in output:
+            test.fail('Secure boot is not enabled. The actual output is %s'
+                      % output)
+
+        # Copy Device Guard to guest
+        dgreadiness_host_path = data_dir.get_deps_dir("dgreadiness_v3.6")
```

Comment:
we also have to push this tool into deps, e.g.
        ls ../../deps/iperf/
        cyggcc_s-1.dll  cygwin1.dll  iperf-2.0.12.exe  iperf-2.0.12.tar.gz  iperf-3.1.3.exe  iperf-3.1.3.tar.gz
Just copy the files we need, e.g. DG_Readiness_Tool_v3.6.ps1, then we don't need unzip

In future, if new version comes out, we can still make a copy, and if we have to test all versions, we can add variants in cfg:

      variants:
           - v3.6:
              xxxx
           - v3.7
              xxx
something like that. Just FYI

Reply:
Sorry, I forgot to push the dgreadiness related files to git.Updated,thanks!

Reply:
Never mind, please create a dir under deps, and push files you need into it